### ISSUE TITLE
three kinds of label VS. hyper parameter "task" binary

### ISSUE THREAD

**Issue by HuXiLiFeng** [aksnzhy/xlearn#197]
Hi，i use a data which has three kinds of labels,such as 0,1,2, In xlearn, the hyper parameter “task” only can be "reg" or "binary", i choose “binary” and run the cod, it generates result as usual。 Why i use a dataset with three kinds of label and choose “binary”，there are no errors？ Are the final result reliable?

**Comment by aksnzhy** [comment]
Hi, Thank you .
For now, xLearn can only solve the binary classification (0 or 1), if you have 0, 1, 2, xLearn will treat both 1 & 2 as 1. For multi-classificaiton problem , you can choose the package like xgboost and lightgbm.
